packages/api-client/src/layouts/Web/ApiClientWeb.vue

```diff
@@ -80,6 +80,11 @@ body {
   overscroll-behavior: none;
 }
 
+body {
+  cursor: default;
+  user-select: none;
```

Comment:
Personally I hate when web apps disable user select since it means I can't copy things. I'd probably prefer to set this in specific places rather than at the body level and risk having users not be able to copy something they otherwise should be able to copy.

Thoughts @cameronrohani?

Reply:
do you have specific example of copyable thing to make sure we do not miss any? i feel like its easier to go that way vs the opposite - what do you think?

i tend to agree however in that client context i feel like it helps giving that native app feeling + copyable infos seems very few / be handled

Reply:
I mean sometimes it can be useful to copy an entire table / set of values. In this PR we can't select any of the response header labels or values or the response body let alone error messages if someone wanted to try Googling those.

I think general consensus it to only disable user select where absolutely necessary. 
Some good responses on the subject: https://ux.stackexchange.com/questions/83184/is-user-select-none-bad-ux

> The goal in employing user-select: none should always be limited to reducing user frustration from unintended selections resulting from interaction that can easily cause them when attempting to carry out the primary task purpose of a control it is used on.

> You should never use user-select: none to prevent selection of content text. The browser itself is responsible for deciding between scrolling and selection in this case and should be relied upon to do its job properly in discerning user intent between the two potential actions (selection or scrolling). While there will always be exceptions, keep in mind that these two actions are globally consistent expected interactions. Interfering with them will likely lead to your application not behaving in a manner that is expected, which is a primary source of frustration.

Reply:
each headers are clickable and redirect to definition, however we should allow their values to be copyable yes! that's a good example!

Reply:
no strong opinion anyway - the main point was to reduce all these unneeded selectable elements and felt easier to treat this way as we allow copy to most of the things + mostly have inputs within the app - so not against dropping it too if we feel that way

Reply:
seems like there could potentially be some a11y & UX implications here as brynn has pointed out so we should prioritize that and evaluate it on a case by case basis if need be? :) Cool with both of you? 😄

Reply:
don't think much impact on a11y as it's only a css property so i guess it should not impact that part but will drop 👍